spring-cloud-stream-binder-kinesis-docs/src/main/asciidoc/overview.adoc

```diff
@@ -89,12 +89,12 @@ headers::
   The set of custom headers to transfer over AWS Kinesis
 +
 Default: "correlationId", "sequenceSize", "sequenceNumber", "contentType", "originalContentType".
-describeStreamBackoff::
-  The amount of time in milliseconds in between retries for the `DescribeStream` operation
+listShardsBackoff::
```

Comment:
Do we really need these separate properties for the list?
Why we can't use existing for "describe" ?

Reply:
Yes, will reuse the old one!